src/swtpm/mainloop.h

```diff
@@ -60,6 +60,7 @@ struct mainLoopParams {
     uint32_t locality_flags;
     TPMLIB_TPMVersion tpmversion;
     uint16_t startupType; /* use TPM 1.2 types */
+    uint32_t lastCommand; /* last Command sent to TPM */
```

Comment:
So this state is still not transferred during migration? I am afraid it is required for proper support.

Reply:
Can you explain why because I don't 'see' it.

Reply:
If the machine shuts down before migration (or saved), reaches target and boot, you don't have the state saved.

Either you have a useful state that must be saved, or you don't and then you don't need it at all.